scripts/modules/b2b-account/quotes.js

```diff
@@ -17,12 +17,12 @@ define([
     "modules/views-paging",
     'modules/editable-view',
     "modules/models-quotes",
-    "modules/b2b-account/account-address-search"], 
+    "modules/b2b-account/account-address-search",
+    "widgets/mz-search-pagination-dd"], 
     function ($, api, _, Hypr, Backbone, HyprLiveContext,
         CustomerModels, CartModels, B2BAccountModels, ProductModalViews,
         ProductPicker, ProductModels, WishlistModels, MozuGrid, MozuGridCollection,
         PagingViews, EditableView, QuoteModels, B2bContactsModal) {
```

Comment:
You added an import but didn't define a name. I understand right now it's a jQuery function but you should define the import here so the number of parameters match.

Reply:
Done